airc/README.md

```diff
@@ -0,0 +1,55 @@
+# AirScript Compiler
+
+This crate aggregates all components of the AirScript compiler into a single place. Specifically, it re-exports functionality from the [parser](../parser/), [ir](../ir/), and [codegen-winter](../codegen-winter/) crates. Additionally, when compiled as an executable, this crate can be used via a [CLI](#command-line-interface-cli) to transpile AIRs defined in AirScript to a specified target language.
+
+## Basic Usage
+
+An in-depth description of AirScript is available in the full AirScript [documentation](https://0xpolygonmiden.github.io/air-script/).
+
+The compiler has three stages, which can be imported and used independently or together.
+
+1. [Parser](../parser/): scans and parses AirScript files and builds an AST
+2. [`AirIR`](../ir/): produces an intermediate representation from an AirScript AST
```

Comment:
nit: "Parser" above is not in quotes but "AirIR" is. We should probably keep it consistent. I slightly prefer the version without quotes.